vcd/resource_vcd_org_vdc.go

```diff
@@ -697,6 +705,16 @@ func getUpdatedVdcInput(d *schema.ResourceData, vcdClient *VCDClient, vdc *govcd
 		vdc.AdminVdc.VmDiscoveryEnabled = d.Get("enable_vm_discovery").(bool)
 	}
 
+	if d.HasChange("elasticity") {
+		elasticityPt := d.Get("elasticity").(bool)
+		vdc.AdminVdc.IsElastic = &elasticityPt
+	}
+
+	if d.HasChange("include_vm_memory_overhead") {
+		memoryOverheadPt := d.Get("include_vm_memory_overhead").(bool)
```

Comment:
Just an option. You may find `takeBoolPointer` useful.

Reply:
changed